### PR TITLE
Replaced network model page with link to CIM profile

### PR DESCRIPTION
**Pull request by ryanjcoleman**
Signed-off-by: ryanjcoleman <ryan.coleman@zepben.com>

### CODE REVIEW

**Comment by ryanjcoleman** [comment]
Task linked: [SITE-76 Urgent - Remove old database doco.](https://app.clickup.com/t/6929263/SITE-76)

**Comment by gcarwardine** [comment]
@ryanjcoleman it may be helpful to put a link in to the JVM SDK stating the the EWB server is built on top of an implementation contained in the SDK.

**Comment by var-ram** [comment]
@ryanjcoleman Also given that we are fixing doco that shouldn't have been there, the same fix should apply to the already released version here: https://github.com/zepben/energy-workbench-server/blob/main/docs/versioned_docs/version-2.6.0/network-model.md

File: docs/docs/network-model.md [hunk 1]
```diff
@@ -3,502 +3,10 @@ id: network-model
...
-- FEEDER\_CB
-- CB
-- FUSE
-- ISOLATOR
-- RECLOSER
-- FAULT\_INDICATOR
-- BROKEN\_BRIDGE
-- GENERATOR
-- CAPACITOR
+If the Evolve profile doesn't contain a part of CIM that you require for your use case, you can request or propose a change to the model following the instructions at: https://github.com/zepben/evolve-data-model
```
Comment (gcarwardine) [inline docs/docs/network-model.md:12]: So I just had a look at https://github.com/zepben/evolve-data-model and we don't actually have any information there - so probably poor place to point people. Can I suggest we either drop this or change it to say to either contact Zepben or start a discussion at the Evolve github discussion: https://github.com/zepben/evolve/discussions

**Review by gcarwardine — CHANGES_REQUESTED** [review]

File: docs/docs/network-model.md (hunk 1 above)
Comment (ryanjcoleman) [reply docs/docs/network-model.md:12]: Sounds good, I copied the paragraph from the SDK documentation so I'll change it there too

**Review by gcarwardine — APPROVED** [review]

**Review by var-ram — APPROVED** [review]

**Pull request merged**